mindjunkies/lecture/tests/test_views.py

```
@@ -91,109 +158,291 @@ def test_lecture_video_forbidden_for_non_enrolled(
-        "create_lecture", kwargs={"course_slug": course.slug, "module_id": module.id}
-    )
-    response = client.post(
-        url, {"title": "New Lecture", "description": "Lecture description", "order": 1}
+        "create_lecture",
+        kwargs={"course_slug": course.slug, "module_id": module.id},
     )
+    data = {"title": "New Lecture", "description": "Lecture description", "order": 1}
+    response = client.post(url, data, follow=True)
+    assert response.status_code == 200
+    assert any("Lecture created successfully" in str(m) for m in get_messages(response.wsgi_request))
+    assert response.redirect_chain[-1][0] == f"{reverse('lecture_home', kwargs={'course_slug': course.slug})}?module_id={module.id}"
```

Ruff (0.8.2)

Line too long (132 > 119)

(E501)